environment/src/main/resources/schema/app/20210712114059_CB-13174_increase_securitygroup_length.sql

```diff
@@ -0,0 +1,10 @@
+-- // CB-13174 increase securitygroup length
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE IF EXISTS environment ALTER COLUMN securitygroup_id_knox SET DATA TYPE VARCHAR(4000);
+ALTER TABLE IF EXISTS environment ALTER COLUMN securitygroup_id_default SET DATA TYPE VARCHAR(4000);
```

Comment:
Shall not we also add `@Column(length = 4000)` (with or without `columnDefinition = "VARCHAR"`) for the respective fields of `Environment` entity?